src/rdiff_backup/FilenameMapping.py

```diff
@@ -191,8 +191,6 @@ def isincfile(self):
                 self.inc_timestr = unquote(temp_rp.inc_timestr)
         else:
             result = rpath.RPath.isincfile(self)
-            if result:
-                self.inc_basestr = unquote(self.inc_basestr)
```

Comment:
I feel suspicious about this change. Why are you skipping the unquote here ? Is inc_basestr already quoted ?
Cause I took a look in git blame and those lines was not changed for a while.

Reply:
So was I, that's why I did put first the PR in WiP, so I can check. Everything seems legit, I guess that I changed enough of the code to create a side effect. Basically this branch uses `self.index` as basis (which is unquoted and doesn't need to be unquoted again, hence the existence of `self.quoted_index`), whereas the other branch of the same function (if index is empty) uses the `self.path` variable, which is quoted.

This is this inconsistency that I don't like in the code, and why I'd like to get rid of QuotedRPath altogether; IMHO it should be transparent. But I don't want to change it right now.

Reply:
Oh, and to be clear: `inc_basestr` is set as a side effect of `isincfile`, and because it is created on the basis of `index`, it is already unquoted and shouldn't be unquoted a 2nd time.